client/src/includes/initTagField.ts

```diff
@@ -0,0 +1,29 @@
+import * as $ from 'jquery'
+/** 
+ *This is a function initTagField
+ *@function
+ *@param {number} id - The object id.
+ *@param {string} autocompleteUrl - The 
+ autoComplete Url.
+ *@param {object} options - Other options 
+ passed to initTagField
+ */
+function initTagField(id: number, autocompleteUrl: string, options: object) {
```

Comment:
I think id should be a string here - as it's used to get an element.

Reply:
Done

Reply:
Okay. I've done that.